tests/base/overlapping_coupling_test.C

```diff
@@ -678,7 +678,7 @@ private:
     system.current_local_solution = libMesh::NumericVector<libMesh::Number>::build(system.comm());
 
     system.current_local_solution->init(system.n_dofs(), system.n_local_dofs(),
-                                        dof_map.get_send_list(), false,
```

Comment:
We ought to leave the unit tests as-is, I think.  We should get more than enough coverage of the new map API option via internal library code, but we also want to continue supporting the old send_list API option, and after upgrading library internals that pretty much leaves these couple unit tests as the last places giving the old API test coverage.